python/cuml/common/input_utils.py

```diff
@@ -130,6 +126,11 @@ def input_to_cuml_array(X, order='F', deepcopy=False,
             raise ValueError("Error: cuDF Series has missing/null values, " +
                              " which are not supported by cuML.")
 
+    # converting pandas to numpy before sending it to CumlArray
+    if isinstance(X, pd.DataFrame) or isinstance(X, pd.Series):
+        # pandas doesn't support custom order in to_numpy
+        X = np.asarray(X.to_numpy(copy=False), order=order)
```

Comment:
Would it give us any performance benefit here to use `cupy.asarray` instead of numpy?